src/pipeline_migration/actions/migrate/cli.py

```diff
@@ -52,7 +230,13 @@ def action(args) -> None:
     else:
         resolver_class = DecentralizationTransitionResolverProxy
 
-    if args.upgrades_file:
+    if args.new_bundles:
+        pipeline_files = args.pipeline_files or search_pipeline_files()
+        if not pipeline_files:
+            return
```

Comment:
Should we log at warning level here someting about that no pipeline_file was found and process is skipped?